src/components/footer/footer.scss

```diff
@@ -0,0 +1,34 @@
+@import '~theme/base/variables';
+@import '~theme/base/mediaqueries';
+@import '~theme/base/utils';
+
+.footerMain {
+  width: 100%;
+}
+
+.footerInner {
+  padding: 1em 0;
+  border-top: 1px solid $color-grey-medium;
+  display: flex;
+  flex-flow: row;
```

Comment:
Isn't `flex-direction: row` by default?

Reply:
Yes 😁, good catch, it's a leftover from trying to make this mobile-first and then reverting back.

Reply:
No worries! :)